File: app/src/withoutPlayServices/java/com/anthony/deepl/openl/util/FirebaseManager.java
package com.anthony.deepl.openl.util;

import android.content.Context;
import android.os.Bundle;
import android.support.annotation.NonNull;

import timber.log.Timber;

public class FirebaseManager implements FirebaseManagerInterface {

    @SuppressWarnings("unused")
    public FirebaseManager(Context context) {}

    @Override
    public void fetchRemoteConfigValues() {}

    @Override
    public void logEvent(String event, Bundle bundle) {}

    @Override
    public Timber.Tree getProductionTimberTree() {
        return new Timber.Tree() {
            @Override
            protected void log(int priority, String tag, @NonNull String message, Throwable t) {
            }
        };
    }
}

File: app/src/withoutPlayServices/java/com/anthony/deepl/openl/util/FirebaseManager.kt
package com.anthony.deepl.openl.util

import android.os.Bundle

import timber.log.Timber

class FirebaseManager : FirebaseManagerInterface {

    override val productionTimberTree: Timber.Tree
        get() = object : Timber.Tree() {
            override fun log(priority: Int, tag: String?, message: String, t: Throwable?) {}
        }

    override fun fetchRemoteConfigValues() {}

    override fun logEvent(event: String, bundle: Bundle?) {}
}
